src/adapter/src/client.rs

```diff
@@ -389,7 +393,7 @@ impl SessionClient {
 
     /// Cancels the query currently running on another connection.
     pub fn cancel_request(&mut self, conn_id: ConnectionId, secret_key: u32) {
-        self.inner_mut().cancel_request(conn_id, secret_key)
+        self.inner_mut().cancel_request(conn_id.val(), secret_key)
```

Comment:
Apparently we have no cancellation test currently?

Reply:
We definitely do! Is it not getting picked up by coverage for some reason?

Reply:
https://github.com/MaterializeInc/materialize/blob/613026cac4c426fe0a78528680228f991a2c20ec/src/environmentd/tests/server.rs#L329-L331

Reply:
Ah! Indeed. That outer function is dead code. The inner function _is_ properly tested. Good spot.

I put up https://github.com/MaterializeInc/materialize/pull/19587 to remove the dead code.